### PR TITLE
Fixed scrolling in eventMainFragment & ScheduleEditFragment

### PR DESCRIPTION
**Pull request by vquelque**
Fixed the scrolling in eventMainFragment (#277) and fixed ScheduleEditFragment which prevented app from starting

### CODE REVIEW

File: app/src/main/java/ch/epfl/sweng/eventmanager/ui/event/interaction/fragments/schedule/ScheduleEditFragment.java [hunk 1]
```diff
@@ -25,6 +26,7 @@
 
 public class ScheduleEditFragment extends Fragment {
     private static final String TAG = "ScheduleEditFragment";
+    private static final String SHEDULE_ITEM = "schedule_item";
```
Comment (antoine-crettenand) [inline app/src/main/java/ch/epfl/sweng/eventmanager/ui/event/interaction/fragments/schedule/ScheduleEditFragment.java]: Forgot a C here - S*C*HEDULE_ITEM

File: app/src/main/java/ch/epfl/sweng/eventmanager/ui/event/interaction/fragments/schedule/ScheduleEditFragment.java [hunk 2]
```diff
@@ -12,6 +12,7 @@
 import androidx.constraintlayout.widget.ConstraintLayout;
 import androidx.fragment.app.Fragment;
 import androidx.lifecycle.ViewModelProviders;
+import androidx.room.Ignore;
```
Comment (antoine-crettenand) [inline app/src/main/java/ch/epfl/sweng/eventmanager/ui/event/interaction/fragments/schedule/ScheduleEditFragment.java]: Why this import ?

File: app/src/main/java/ch/epfl/sweng/eventmanager/ui/event/interaction/fragments/schedule/ScheduleEditFragment.java (hunk 1 above)
Comment (vquelque) [reply app/src/main/java/ch/epfl/sweng/eventmanager/ui/event/interaction/fragments/schedule/ScheduleEditFragment.java]: Thanks for spotting !

File: app/src/main/java/ch/epfl/sweng/eventmanager/ui/event/interaction/fragments/schedule/ScheduleEditFragment.java (hunk 2 above)
Comment (vquelque) [reply app/src/main/java/ch/epfl/sweng/eventmanager/ui/event/interaction/fragments/schedule/ScheduleEditFragment.java]: Don't know why It didn't self optimized imports  ;)

**Review by antoine-crettenand — DISMISSED** [review]

**Review by zyuiop — DISMISSED** [review]

**Review by zbirobin — DISMISSED** [review]

**Review by antoine-crettenand — APPROVED** [review]

**Review by zyuiop — APPROVED** [review]

**Review by zbirobin — APPROVED** [review]

**Pull request merged**